src/components/VolumeIndicator/VolumeIndicator.vue

```diff
@@ -102,45 +105,56 @@ export default {
 				return 0
 			}
 
-			return this.size * (1 - this.currentVolume / this.volumeThreshold)
+			return this.size * this.computeVolumeLevel(20)
 		},
+	},
 
-		overlayMutedColor() {
-			return !this.audioEnabled ? this.overlayColor : undefined
+	methods: {
+		/**
+		 * Compute volume proportion based on currentVolume [-100, 0]
+		 * and volumeThreshold. Returns number between [0, 1]
+		 *
+		 * @param {number} overload to visually increase volume level
+		 */
+		computeVolumeLevel(overload = 0) {
+			const computedLevel
+				= (this.volumeThreshold - this.currentVolume)
+				/ (this.volumeThreshold + overload)
+
+			if (computedLevel < 0) return 0
+			else if (computedLevel > 1) return 1
+			else return computedLevel
 		},
 	},
 }
 </script>
 
 <style lang="scss" scoped>
-.volume-indicator {
+.volume-indicator-wrapper {
 	position: relative;
 }
 
-.volume-indicator-primary {
+.volume-indicator {
 	position: absolute;
-	top: 0;
 	left: 50%;
 	transform: translateX(-50%);
 
+	width: 100%;
+	height: 100%;
+
 	overflow: hidden;
+
+	transition: height 0.2s linear;
```

Comment:
added `transition` to smoothly change the indicator

Reply:
Currently, the indicator is made by two clipped icons on the top and the button with heights:
- `currentVolumeIndicatorHeight`
- `size - 2 - currentVolumeIndicatorHeight`

This `-2` emulates a 2px gap like a border. But with transition, sometimes it has an incorrect size. Sometimes the gap is large than 2px, and sometimes, there is no gap.

Instead of making this gap by the size difference, maybe add an actual border?

```css
.volume-indicator-overlay {
  border-top: 2px solid var(--color-main-background);
}
```

Reply:
The component implements in three places:

- MediaDevicesPreview - bg color is `var(--color-main-background)`
- DeviceChecker - bg color is `var(--color-main-background)`
- LocalMediaControls - bg color is image and `rgba(34, 34, 34, 0.8)` over it. So, considering that this is the mostly visible place of this component, actual border won't fit here

Reply:
Ho, yes, you are right. A CSS border doesn't work in `LocalMediaControls`...

But the current solution sometimes looks buggy...

| Normal | No gap | Large gap |
|---|---|---|
| ![image](https://user-images.githubusercontent.com/25978914/214841992-ca7e70a0-d4f4-4475-98ca-15984a3df25a.png) | ![image](https://user-images.githubusercontent.com/25978914/214841915-3b0f2b66-6480-447a-8cf3-82357fe203a5.png) | ![image](https://user-images.githubusercontent.com/25978914/214841705-902acb44-e8c7-4289-8294-90a8bff8b726.png) |

GIF:

![mic-bug](https://user-images.githubusercontent.com/25978914/214842048-6ac62f5c-59f0-417f-94fe-e001cdfb3f40.gif)

Reply:
*Thinks... 🤔 *